cores/cv32e40p/user_manual/source/corev_hw_loop.rst

```diff
@@ -1,7 +1,7 @@
 .. _hwloop-specs:
 
-PULP Hardware Loop Extensions
-=============================
+CORE-V Hardware Loop Extensions
```

Comment:
> 
> 
> @pietraferreira, thanks for all the updates. There are some inconsistencies in the name (not due to your specific pull request but now it's more visible).
> We have the RTL parameter called PULP_XPULP selecting whether the custom extensions are enabled or not. In the compiler they are called Xcorev. In the user manual they are called PULP extensions, now CORE-V extensions. We should agree how to call them consistenlty. Nothing wrong of you. Just the parameter is called PULP and the extensions are called XCOREV. Asking also @jm4rtin and @Silabs-ArjanB' opinions on that.
> 
> Another thing, the bit manipulation and SIMD extensions are not implemented in the CORE-V compiler. I know. But now the user reading the USER MANUAL sees some instructions cv.something, and others pv.somethings or p.something. We should be consistent with all of these instructions, and saying that they are not automatically generated by the compiler at the moment, or something like this. But they are still part of the ISA and still can be execued by the core writing exadecimal numbers ... They will show up in the core tracer as "pv" or "cv" something. So that part of the user manual should be updated too.

I would prefer greatly if we do not change the actual parameter names.

Reply:
The external naming for OpenHW Group needs to move to CORE-V, if nothing else because I understand that the PULP project is separately discussing upstreaming its own compiler, which would use `Xpulp`.  However this does not affect what is used internally within the RTL.

The PULP bitmanip and SIMD instructions are problematic, since my understanding is that CORE-V will move to the upstream B and P extensions.  I'd really like to see them removed.  However while they remain in the RTL code base, the information probably needs to be there. Perhaps they do into an appendix?

Reply:
Hi @jeremybennett . As for today, the "cv32e40p" core won't change its ISA. So the current version of the bit manipulation and packed simd extensions won't change (the P and B won't be retified any soon by RISC-V afaik). 
What we need to do imo is to say that they are not supported by the core-v gcc compiler, but the RTL implements them. So if any of the members (as e.g., @gtagliavini or @eflamand - please add your inputs as well) want to implement them in the compiler or use assembly with hexadecimal numbers, they should be able to. Same, if any member wants to extend LLVM to support these instructions, they should be able to do it too.
Thus, for consistency, the instructions should be listed as cv. as they are part of the core-v cv32e40p core today.
@Silabs-ArjanB , @jm4rtin agree we this?

Reply:
@davideschiavone I'm happy to do this if @Silabs-ArjanB and @jm4rtin are happy. We can make it clear there is not currently compiler support.

@pietraferreira - can you add these changes if @Silabs-ArjanB and @jm4rtin are OK with them.

Reply:
Hi @jeremybennett @davideschiavone I am fine if the PULP B-like and P-like instructions are also prefixed with cv. as long as **if** we support such instructions in the CORE-V toolchain there is also an option to include/exclude such groups of instructions with an appropriate level of granularity (e.g. include/exclude all PULP P-like instructions, include/exclude hardware loop instructions, etc.)

Reply:
If @jeremybennett agrees, we can do something like this.
For each subset of Xcorev instructions, we write:
This subset of instructions (e.g. hwloop/branch immediate) is enabled in the compiler by Xcorevhwloop/Xcorevbranchimm etc.
In the SIMD and BitMan, we write "they are not supported by the compiler"

Reply:
@Silabs-ArjanB @davideschiavone No problem. The feature is already there. You currently have `-Xcorev` for all supported extensions and `-Xcorevhwlp`, `-Xcorevmac` etc to select specific subsets.  This should be documented in the relevant tool manuals (which is where this sort of thing belongs).